### DISCUSSION TITLE
Why most blazor nuget packages do not work on MAUI Blazor?

### DISCUSSION THREAD

**Discussion by nextcodelab** [dotnet/maui#6411] [Q&A]
I try many blazor nuget packages like this [https://github.com/Blazored/Modal](https://github.com/Blazored/Modal).
Almost all of those packages do not work on Blazor Maui.

Why blazor is not made as universal so we use it everywhere?

I also tried this on Maui Blazor but not working.. [https://www.matblazor.com/](https://www.matblazor.com/)

**Comment by MihirJayavant** [comment]
I tried Matblazor with Blazor Maui and it is working fine.
